src/Commands/DiscoverChildren.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Tightenco\Parental\Commands;
+
+use hanneskod\classtools\Iterator\ClassIterator;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Symfony\Component\Finder\Finder;
+use Tightenco\Parental\HasChildren;
+use Tightenco\Parental\HasParent;
+
+class DiscoverChildren extends Command
+{
+    protected $signature = 'parental:discover-children';
+
+    protected $description = 'Discover the child models of parent classes using the HasChildren trait.';
+
+    public function handle()
+    {
+        if (! file_exists(config_path('parental.php'))) {
+            file_put_contents(config_path('parental.php'), file_get_contents(__DIR__.'/../../config/parental.php'));
+        }
+
+        $content = file_get_contents(config_path('parental.php'));
+        $asString  = "'discovered_children' => ".var_export($this->findChildren(), true).', //just an anchor';
+
+        $content = preg_replace("/\'discovered_children\' => .*, \/\/just an anchor/", $asString, $content);
+
+        file_put_contents(config_path('parental.php'), $content);
```

Comment:
not comfortable editing the config file from a command honestly.

Reply:
Yes, me neither, I'm still not 100% sure it's working tbh - I'm terrible with regex... I should probably just store the child classes in a dedicated file to avoid cluttering the config file.